recipes/onnxruntime/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+python tools/ci_build/build.py --build_dir build/Linux --config Release \
```

Comment:
Could you please use ``${PYTHON}`` instead?
```suggestion
python tools/ci_build/build.py --build_dir build/Linux --config Release \
```